### PR TITLE
Update section-3.mdx

### PR DESCRIPTION
**Pull request by ridetoruin**
Have added back ticks to the right parts to bring formatting in-line with other docs.

Thank you for wanting to contribute to sPhil! 🧙 🦉

We would like to keep our code and writing as neat and tidy as possible, and would appreciate if you could verify the following. Each PR should _either_ contribute content or functional code.

## Checklist (pick `Docs` or `Code`)

- [x] Docs contribution (philosophy, literature, content)
    - [x] **REQUIRED** I accept that my writing is submitted under the [ATTRIBUTION-NONCOMMERCIAL-SHAREALIKE 4.0 INTERNATIONAL](https://creativecommons.org/licenses/by-nc-sa/4.0/), which, briefly put, prohibits commercial re-use of the content, but allows sharing, remixing and building upon the material insofar as attribution is given (see [legal code](https://creativecommons.org/licenses/by-nc-sa/4.0/legalcode.en)), and in this regard I understand that my writing is open to be modified, remixed and build upon by others in the sPhil community within the `systemphil/sphil` project in future perpetuity.
    - [x] **REQUIRED** I have followed the [formatting guidelines](https://github.com/systemphil/sphil/blob/dev/src/pages/contributing/formatting/basic-markdown.md).
    - [x] **REQUIRED** I have followed the [MLA citation style](https://owl.purdue.edu/owl/research_and_citation/mla_style/mla_formatting_and_style_guide/mla_formatting_and_style_guide.html).
    - [x] **REQUIRED** I have added or verified title and description metadata at the very top of the file. Example:

        ```md
        ---
        title: Hegel Guides
        description: Learn about the philosophy of G. W. F. Hegel
        ---
        ```

    - [ ] My article is a stub or I want to actively encourage contribution, I've added the Stub component to the bottom of my content but _before_ the bibliography:

        ```ts
        import Stub from "@/components/Stub";

        <Stub />
        ```

    - [x] **REQUIRED** I have added, verified or extended a bibliography. This is added before the authors section and follows MLA guidelines. Example:

        ```mdx
        ## Works Cited

        <div className="text-sm">
        - Hegel, Friedrich Georg Wilhelm. _The Science of Logic_. Translated by George di Giovanni, Cambridge University Press, 2010.
        - Kant, Immanuel. _The Critique of Pure Reason_. Edited by Paul Guyer and Allen W. Wood, Cambridge University Press, 1998.
        </div>
        ```

    - [x] I have signed the document with my name/username under either as `Authors`, `Editors` or `Contributors`. 
        > Use **Authors** if you have created and substantially added content.  
          Use **Editor** if you have made substantial edits or review.  
          Use **Contributor** if you have made minor edits, reviews or contributions.  
          If you've done multiple, pick the most weighted: Author > Editor > Contributor.  
          If you prefer to remain anonymous, that's fine too, but note that a record of your contributions based on your GitHub username will exist here in the codebase.

        ```md

        // Bibliography goes here

        ---

        **Authors**  
        Ahilleas Rokni (2024), Tom Bombadil (2025)

        **Contributors**  
        Filip Niklas (2024), Boromir (2025)
        ```

    - [ ] If the article makes use of footnotes, I have checked that `**Notes**` is exactly the very last item in the file.

- [ ] Code contribution ([Apache version 2 license](https://www.apache.org/licenses/LICENSE-2.0.txt))
    <details>

    All code apart of what is inside `src/pages/**` (excluding `/contributing/**`, `_app.mdx`, `_document.tsx`, `_meta.json`, `acknowledgements.mdx`, `index.mdx`, `privacy.mdx`, `team.mdx`, `terms.mdx`) is subject to Apache version 2 license. Basically, anything outside of content, literature, philosophy.

    </details>

### CODE REVIEW

File: src/pages/hegel/reference/mechanical-object/section-3.mdx [hunk 1]
```diff
@@ -3,20 +3,21 @@ title: Section 3
...
+In Section 3 of `A. The Mechanical Object`, Hegel builds on the analysis of the infinite regress of determinism, because, as Hegel will go on to show, determinism leads to a contradiction that must be resolved. It is this contradiction that will lead us to the next chapter of Mechanism.
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/section-3.mdx:6]: Empty line needed to separate paragraphs.

**Review by Firgrep — CHANGES_REQUESTED** [review]
Please review the checklist and make sure everything is accounted for.

From a quick glance I can see there are no headings, bibliography, authors, MLA citation style... 

If each of these sections are only going to be like 5-6 paragraphs, you may want to consider combining them to a single article so that you don't have to re-add elements like bibliography, authors, etc.

File: src/pages/hegel/reference/mechanical-object/section-3.mdx (hunk 1 above)
Comment (ridetoruin) [reply src/pages/hegel/reference/mechanical-object/section-3.mdx:6]: done

**Comment by ridetoruin** [comment]
Alright, I think I'm good!

**Comment by Firgrep** [comment]
Looking much better! Just add a heading inside of the article (remember to use ## instead of # )

File: src/pages/hegel/reference/mechanical-object/section-3.mdx [hunk 2]
```diff
@@ -1,22 +1,36 @@
...
+>“This contradiction is, therefore, the *negative unity* of a number of objects which, in that unity, simply repel one another: this is the *mechanical process*.” (Hegel, 1991, 713)
 
+The conclusion of the `A. The Mechanical Object`, then, is that these mechanical objects that were initially taken as immediately identical, indeterminate, and indifferent, are in fact in a contradiction with each other. This contradiction is a negative unity whereby what it is for a `mechanical object` to be itself is for its determinateness to be identical to the determinateness of other mechanical objects (the *unity* of the negative unity), whilst for it to be indifferent to other mechanical objects (the *negative* in negative unity. This new unity takes us into the next chapter of `Mechanism`, `B. The Mechanical Process`.
```
Comment (Firgrep) [inline src/pages/hegel/reference/mechanical-object/section-3.mdx:22]: Also, I don't think you should be using back ticks for section titles, as these aren't technical terms per say but textual. What do you think?

Comment (Firgrep) [reply src/pages/hegel/reference/mechanical-object/section-3.mdx:24]: I think for textual headings we should just capitalize it normally without the backticks?

**Review by Firgrep — APPROVED** [review]
Looking good now. I've

- removed quotation marks from block quotes, shuffled the period to after the citation. 
- updated metadata description
- replaced comma with colon in content title
- removed some single right quotation marks lurking around

If this looks good to you, you can merge it.

**Pull request merged**